first commit:

leins275 opened the pull request:
modified template from vdv proj

Commit:
first commit:
modified template from vdv proj

daniilsavchuk commented on .idea/EACH.iml:
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Please add '.idea' directory to the .gitignore and **amend** the current commit

daniilsavchuk commented on Backend/DB-scripts/CreateBase.sql:
```diff
@@ -0,0 +1,230 @@
+DROP SEQUENCE IF EXISTS each_seq;
```
Please drop this file

daniilsavchuk commented on Backend/VERSION:
```diff
@@ -0,0 +1 @@
+0.0.1
```
Please add "new line" at the end of the file

daniilsavchuk commented on Backend/config.json:
```diff
@@ -0,0 +1,112 @@
+{
```
Please drop all profiles that currently useless. Keep only local and dev. Thank you in advance!

daniilsavchuk commented on Backend/each/Entities/EntityBase.py:
```diff
@@ -0,0 +1,88 @@
+import json
```
Please drop all entities classes

daniilsavchuk commented on Backend/each/Prop/PropBase.py:
```diff
@@ -0,0 +1,84 @@
+from collections import OrderedDict
```
Please drop all Prop classes

daniilsavchuk commented on Backend/each/app.py:
```diff
@@ -0,0 +1,1212 @@
+from each.db import DBConnection
+from each.serve_swagger import SpecServer
+
+from each.Entities.EntityBase import EntityBase
```
Please remove all unused IMPORTS after code cleanup

daniilsavchuk commented on Backend/each/app.py:
```diff
@@ -0,0 +1,1212 @@
+    resp.body = obj_to_json([o.to_dict() for o in objects])
+    resp.status = falcon.HTTP_200
+
+def getCourtById(**request_handler_args):
```
Please drop all useless functions

daniilsavchuk commented on Backend/each/auth/config.py:
```diff
@@ -0,0 +1,47 @@
+class PROVIDER:
```
Please drop this useless file

daniilsavchuk commented on Backend/each/search.py:
```diff
@@ -0,0 +1,79 @@
+from collections import OrderedDict
```
Please drop this useless file

daniilsavchuk commented on Backend/swagger_temp.json:
```diff
@@ -0,0 +1 @@
+{"swagger": "2.0", "info": {"version": "2.0.1", "title": "EACH"}, "host": "127.0.0.1:4201", "basePath": "/each", "tags": [{"name": "default", "description": "empty default feature"}], "schemes": ["http"], "paths": {}, "definitions": {}, "externalDocs": {}}
```
please add this temporary file to .gitignore and **amend** this from commit

daniilsavchuk commented on Backend/tests:
```diff
@@ -0,0 +1,22 @@
+{
```
Please drop this outdated for our application file

daniilsavchuk reviewed (changes requested):
Looks really great, you are on the right way, please keep doing that. 

Some additional imporvements required

Commit:
fixed issues:

 - added .gitignore file (.idea dir and swagger_temp.json file added)

  - dropped files:
  -- Backend/tests
  -- Backend/each/search.py
  -- Backend/each/auth/
  -- Backend/each/Entities/
  -- Backend/each/Prop/
  -- Backend/DB-scripts/CreateBase.sql

  - deleted all profiles except local_and_dev

  - new line added to Backend/VERSION file
